lib/registries/addon/components/sharing-icons/popover/template.hbs

```diff
@@ -0,0 +1,14 @@
+
+<div class="dropdown pull-right">
+    {{#bs-dropdown as |dd|}}
+        {{#dd.button class='btn-link' local-class='SharingIconsPopover__button'}}
+            <span class="fa fa-share-alt"></span>
+        {{/dd.button}}
+
+        {{#dd.menu class='dropdown-menu' local-class='SharingIconsPopover'}}
+            {{yield}}
+        {{/dd.menu}}
+    {{/bs-dropdown}}
+</div>
+
+{{!-- ember bootstrap popover to see if it likes me more --}}
```

Comment:
this line needs to go away. Should try using bootstrap popover instead of dd, because bad.